notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -0,0 +1,512 @@
+    "    !pip install git+https://github.com/openvinotoolkit/nncf.git#egg=nncf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d6b41e6-132b-40da-b3b9-91bacba29e31",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "771388d6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/root/akozlov/virt_envs/ov_notebooks/lib/python3.8/site-packages/nncf/torch/__init__.py:23: UserWarning: NNCF provides best results with torch==1.9.1, while current torch version is 1.13.1+cu117 - consider switching to torch==1.9.1\n",
+      "  warnings.warn(\"NNCF provides best results with torch=={bkc}, \"\n",
+      "/root/akozlov/virt_envs/ov_notebooks/lib/python3.8/site-packages/nncf/torch/dynamic_graph/patch_pytorch.py:163: UserWarning: Not patching unique_dim since it is missing in this version of PyTorch\n",
+      "  warnings.warn(\"Not patching {} since it is missing in this version of PyTorch\".format(op_name))\n",
+      "/root/akozlov/virt_envs/ov_notebooks/lib/python3.8/site-packages/openvino/offline_transformations/__init__.py:10: FutureWarning: The module is private and following namespace `offline_transformations` will be removed in the future, use `openvino.runtime.passes` instead!\n",
+      "  warnings.warn(\n",
+      "/root/akozlov/virt_envs/ov_notebooks/lib/python3.8/site-packages/transformers/generation_utils.py:24: FutureWarning: Importing `GenerationMixin` from `src/transformers/generation_utils.py` is deprecated and will be removed in Transformers v5. Import as `from transformers import GenerationMixin` instead.\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "import time\n",
+    "from functools import partial\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from datasets import load_dataset\n",
+    "from transformers import AutoModelForSequenceClassification, AutoTokenizer\n",
+    "\n",
+    "from optimum.intel.openvino import OVQuantizer\n",
+    "from optimum.intel.openvino import OVConfig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7603a481",
+   "metadata": {},
+   "source": [
+    "## Quantize model with HuggingFace Optimum API\n",
+    "The sparsity acceleration MatMul operations is available only in the case when these operations are quantized into 8-bit precision. If the model is not quantized it can be done using either way availble for OpenVINO models. For more details refer [here](https://docs.openvino.ai/latest/openvino_docs_model_optimization_guide.html). In this tutorial we use the HuggingFace Optimum API to quantize the model. The HuggingFace Optimum API is a high-level API that allows to convert and quantize models from the HuggingFace Transformers library to the OpenVINO™ IR format. For more details refer to the [HuggingFace Optimum documentation](https://huggingface.co/docs/optimum/intel/optimization_ov)."
```

Comment:
```suggestion
    "The sparsity acceleration MatMul operations are only available cases when these operations are quantized into 8-bit precision. If the model is not in 8-bit precision, it can be quantized using either method available for OpenVINO models. For more details, please refer to the [Model Optimization Guide](https://docs.openvino.ai/latest/openvino_docs_model_optimization_guide.html). In this tutorial we use the Hugging Face Optimum API to quantize the model. The Hugging Face Optimum API is a high-level API that allows us to convert and quantize models from the Hugging Face Transformers library to the OpenVINO™ IR format. For more details refer to the [Hugging Face Optimum documentation](https://huggingface.co/docs/optimum/intel/optimization_ov)."
```